commons/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="free.nrw.commons"
```

Comment:
Hum... package names can be anything, there is no reason that one would work better than the other... have you run a full-text search for the older names to see if one subsists? A grep -R on Linux or grep -r on Mac... by the way what OS are you using?